runelite-client/src/main/java/net/runelite/client/util/StackFormatter.java

```diff
@@ -61,6 +65,36 @@
 		DecimalFormatSymbols.getInstance(Locale.ENGLISH)
 	);
 
+	/**
+	 * Attempts to call the platform to get a localized time string based on
+	 * the users preferences. Falls back on using locale default if it is on a
+	 * platform that has no consistent way of obtaining this information.
+	 *
+	 * @param localDateTime The LocalDateTime object to format as a string
+	 * @return The formatted string.
+	 */
+	public static String getPlatformTimeStringFromLocalDateTime(LocalDateTime localDateTime)
```

Comment:
This has nothing to do with stack formatting, it should be somewhere else. The method name is too long; don't include types in it, that is what the method signature is for.

Reply:
I would maybe rename the class to formatter. Or make TimeUtil with 1 method? But I like just formatter